src/assets/legwear/leggings/leggings.asset.ts

```diff
@@ -0,0 +1,51 @@
+DefineAsset({
+	name: 'Leggings',
+	size: 'small',
+	graphics: 'graphics.json',
+	colorization: {
+		leggings: {
+			name: 'Leggings',
+			default: '#2A2A2A',
+		},
+	},
+	// size:800, y:574, centered
+	preview: 'preview.png',
+	attributes: {
+		provides: [
+			'Clothing',
+			'Legwear',
```

Comment:
I guess this is a matter definition and not really about right/wrong. Just making sure you did it intentionally.

I could not find the leggings, as I was looking in the trousers, pants, and skirts section which is `Clothing_lower`, not in the `Legwear section`, which means socks, stockings, and pantyhoses. 

The intended difference was things worn mainly over just the legs, but potentially going over the hops, but primarily "underwear", whereas everything that can be worn over the hips and down the legs but is a primary clothing on the outside, is the other section.

Reply:
Moved it to the bottom category for consistancy